storage/src/persistent/commit_log.rs

```diff
@@ -62,7 +62,7 @@ impl slog::Value for CommitLogError {
 }
 
 type ByteLimit = usize;
-type ItemCount = u16;
+type ItemCount = u32;
```

Comment:
Please increase `DATABASE_VERSION` in `main.rs` to indicate that format of the data in persistent storage has changed.

Reply:
Has it changed, though? The ItemCount is only used to read from the database:

`fold_consecutive_locations()` uses Range, which uses ItemCount,
and `fit_batch_read_limit()` uses it as a parameter.

I increased the size, because it was causing overflow in `fold_consecutive_locations`, but if I understand it correctly, the data format hasn't changed? 🤔

Reply:
Yes, you are right. For some reason I remembered it being stored but that is not the case here.